react exercise/users-information/src/components/creator-card/CreatorCard.scss

```diff
@@ -0,0 +1,121 @@
+$number-of-posts: 3;
+$creator-card-width: 342px;
+$creator-profile-size: 80px;
+$post-preview-width: $creator-card-width / $number-of-posts;
+$profile-offset-from-post: ($post-preview-width - $creator-profile-size) / 2; /* Center the profile image */
+
+.creator-card {
+    display: flex;
+    width: $creator-card-width;
+    height: 416px;
+    min-width: 296px;
+    padding-bottom: 32px;
+    flex-direction: column;
+    position: relative;
+    gap: 12px;
+
+    border-radius: 16px;
+    background: #FFF;
+    box-shadow: 0px 4px 12px -2px rgba(24, 39, 75, 0.08), 0px 0px 4px 0px rgba(24, 39, 75, 0.02);
+    
+    /* Dummy styling for posts-preview */
+    .creator-posts-container {
+        display: flex;
+        justify-content: space-between;
+      
+        .creator-post {
+          flex: 1 1 $post-preview-width; /* Allow growing and shrinking with a base size */
+          min-width: $post-preview-width; /* Prevent shrinking below the base width */
```

Comment:
Since I needed to calculate the profile-offset-from-post using the post-preview-width, I already calculated it and felt it is more clear to specify it explicitly, but it is actually redundant as you mention